lib/github_core.ml

```diff
@@ -2002,6 +2011,45 @@ module Make(Env : Github_s.Env)(Time : Github_s.Time)(CL : Cohttp_lwt.S.Client)
       API.get_stream ?token ~uri
         ~fail_handlers
         (fun b -> return (contributors_stats_of_string b))
+
+    let yearly_commit_activity ?token ~user ~repo () =
+      let uri = URI.repo_commit_activity ~user ~repo in
+      let fail_handlers = [
+          API.code_handler
+            ~expected_code:`Accepted
+            (fun _ -> Lwt.return [])
+        ] in
+      API.get_stream ?token ~uri
+        ~fail_handlers
+        (fun b -> return (commit_activities_of_string b))
+
+    let weekly_commit_activity ?token ~user ~repo () =
+      let uri = URI.repo_code_frequency_stats ~user ~repo in
+      let fail_handlers = [
+          API.code_handler
+            ~expected_code:`Accepted
+            (fun _ -> Lwt.return [])
+        ] in
+      API.get_stream ?token ~uri
+        ~fail_handlers
+        (fun b -> return (code_frequencies_of_string b))
+
+    let weekly_commit_count ?token ~user ~repo () =
+      let uri = URI.repo_code_frequency_stats ~user ~repo in
```

Comment:
I bet this is the issue causing the parse error: you are not using the correct URI for the `weekly_commit_count`. It should be `/repos/:owner/:repo/stats/participation`, not `/repos/:owner/:repo/stats/code_frequency`!

Reply:
Good catch!

Reply:
Yes of course that's right. I'll add another test program to exercise all the stats endpoints I've added.